Userland/Libraries/LibWeb/HTML/Parser/HTMLDocumentParser.cpp

```diff
@@ -2103,7 +2103,7 @@ void HTMLDocumentParser::handle_in_table_text(HTMLToken& token)
             return;
```

Comment:
the commit message says to make it non movalbe, but moves it, I guess you mean non-copyable, like the next commit does